pom.xml

```diff
@@ -723,6 +723,7 @@
                 <banDuplicateClasses>
                   <ignoreClasses>
                     <ignoreClass>META-INF/versions/*/org/bouncycastle/*</ignoreClass>
+                    <ignoreClass>META-INF/versions/*/com/fasterxml/jackson/core/io/doubleparser/*</ignoreClass>
```

Comment:
I'm not sure if we want to keep adding to this list to resolve duplicate class issue, I know that we have a long history of issue for boucycastle. I believe what we did before is to exclude the class from one of the package? cc: @sfc-gh-lsembera

Reply:
@sfc-gh-lsembera any concerns here?